src/routes/profile.svelte

```diff
@@ -32,6 +32,7 @@
     ]));
 
     data.timelineFetchedUntil = timelineFetchedUntil;
+    data.account.csrf_token = session.account.csrf_token;
```

Comment:
Assigning to `data.account.csrf_token` right before checking if `data.account == null`?